packages/dd-trace/src/opentracing/span_context.js

```diff
@@ -43,6 +43,12 @@ class DatadogSpanContext {
     }
   }
 
+  // toTraceIdBigInt returns the traceId as a BigInt
+  // in case of a 128 bit traceId, it returns the lower 64 bits
+  toTraceIdBigInt () {
```

Comment:
This adds a public API. It should either be documented if we expect users to use it, or we should use the ID directly if we prefer to keep this private.